c7n/resources/asg.py

```diff
@@ -1200,6 +1200,19 @@ def process(self, asgs):
             instance_count = sum(list(w.map(self.process_asg, asgs)))
             self.log.info("Applied tags to %d instances" % instance_count)
 
+    @contextmanager
+    def retry_remaining_instances(self, instances):
+        client = local_session(self.manager.session_factory).client('ec2')
+        try:
+            client.describe_instances(InstanceIds=instances)
```

Comment:
doing separate describe calls per batch of instances is definitely not what we want to do here.

Reply:
Working on it! Thanks

Reply:
Took out the describe calls